src/Go.CodeAnalysis/Parser/ImportDeclarationNode.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace Go.CodeAnalysis.Parser
+{
+    using System.Collections.Generic;
+    using System.Collections.Immutable;
+    using Go.CodeAnalysis.Common;
+    using Go.CodeAnalysis.Lex;
+    using Go.CodeAnalysis.Text;
+
+    /// <summary>
+    /// Represents a single package import declaration.
+    /// </summary>
+    public class ImportDeclarationNode : ParseNode
+    {
+        public ImportDeclarationNode(
+            SnapshotSegment code,
+            string packageName,
+            string alias = null)
+            : base(code, ImmutableArray<ParseNode>.Empty)
+        {
+        }
+
+        public string PackageName { get; private set; }
```

Comment:
Two possibilities here: the extent already has this information. We can just use that.. or we can fold this info from this node right into the ImportsNode and just have it have an immutable array of segments.

What do you think?

Reply:
It looks like `ImportNode` could be reused perhaps and we could just add a second TryParse() that parses this permutation?

Reply:
I thought about just keeping an array in ImportNode. It will become tricky when we support alias. Another possibility is to put all import declarations under ImportsNode.